index.html

```diff
@@ -674,26 +709,59 @@
           <a>push subscription</a>. The <a>user agent</a> MAY consider these options when
           requesting <a>express permission</a> from the user. When an option is considered, the
           <a>user agent</a> SHOULD enforce it on incoming <a>push messages</a>.
         </p>
-        <dl title="dictionary PushSubscriptionOptions" class="idl">
+        <p>
+          Once set, options for a <a>push subscription</a> cannot change. A pre-existing <a>push
+          subscription</a> can be <a href=
+          "#widl-PushSubscription-unsubscribe-Promise-boolean">unsubscribed</a> to create a <a>push
+          subscription</a> with new options.
+        </p>
+        <dl title="dictionary PushSubscriptionOptionsDict" class="idl" data-merge=
+        "PushSubscriptionOptions">
           <dt>
             boolean userVisibleOnly = false
           </dt>
+          <dt>
+            BufferSource? applicationServerKey = null
+          </dt>
+        </dl>
+        <dl title="interface PushSubscriptionOptions" class="idl" data-merge=
+        "PushSubscriptionOptionsDict">
+          <dt>
+            readonly attribute boolean userVisibleOnly
+          </dt>
+          <dt>
+            [Throws] readonly attribute ArrayBuffer? applicationServerKey
+          </dt>
         </dl>
         <p>
           The <code><dfn id=
-          "widl-PushSubscriptionOptions-userVisibleOnly">userVisibleOnly</dfn></code> option, when
-          set to <code>true</code>, indicates that the <a>push subscription</a> will only be used
-          for <a>push messages</a> whose effect is made visible to the user, for example by
+          "widl-PushSubscriptionOptionsDict-userVisibleOnly">userVisibleOnly</dfn></code> option,
+          when set to <code>true</code>, indicates that the <a>push subscription</a> will only be
+          used for <a>push messages</a> whose effect is made visible to the user, for example by
           displaying a Web Notification. [[NOTIFICATIONS]]
         </p>
+        <p>
+          The <code><dfn id=
+          "widl-PushSubscriptionOptionsDict-applicationServerKey">applicationServerKey</dfn></code>
+          option includes an elliptic curve public key for an application server. This is the key
+          that the application server will use to authenticate itself when sending push messages to
+          this subscription as defined in [[!WEBPUSH-VAPID]]; the push service will reject any push
+          message unless the corresponding private key is used to generate an authentication token.
+        </p>
+        <p>
+          If present, the value of <code><a>applicationServerKey</a></code> MUST include a point on
+          the P-256 elliptic curve [[!FIPS-186-3]], encoded in the uncompressed form described in
+          [[!X9.62]] Annex A (that is, 65 octets, starting with an 0x04 octet). A different key to
+          the one used for message encryption [[WEBPUSH-ENCRYPTION]] is strongly recommended.
```

Comment:
This MUST be a different key than the one used for message encryption [[WEBPUSH-ENCRYPTION]].